xstream/server/server/rest.go

```diff
@@ -97,12 +98,26 @@ func createRestServer(port int) *http.Server {
 	return server
 }
 
+type information struct {
+	Version       string `json:"version"`
+	Os            string `json:"os"`
+	UpTimeSeconds int64  `json:"upTimeSeconds"`
+}
+
 //The handler for root
 func rootHandler(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close()
 	switch r.Method {
 	case http.MethodGet, http.MethodPost:
 		w.WriteHeader(http.StatusOK)
+		info := new(information)
+		info.Version = version
+		info.UpTimeSeconds = time.Now().Unix() - startTimeStamp
+		info.Os = runtime.GOOS
+		if byteInfo, err := json.Marshal(info); err == nil {
+			w.Write(byteInfo)
+			return
+		}
```

Comment:
Print an error message in log if having any error. Or if it could not happen, just ignore the error check?